src/languages/en-US.ts

```diff
@@ -217,7 +217,8 @@ export default class extends Language {
 		inhibitorRunin: ({ type }) => `This command is only available in ${type} channels.`,
 		inhibitorRuninNone: ({ name }) => `The ${name} command is not configured to run in any channel.`,
 		inhibitorDisabledGuild: 'This command has been disabled by an admin in this guild!',
-		inhibitorDisabledGlobal: 'This command has been globally disabled by the bot owners!',
+		inhibitorDisabledGlobal:
+			'This command has been globally disabled by the bot owners. Want to know why and find out when it will be back? Join the official Skyra server: https://join.skyra.pw',
```

Comment:
```suggestion
			'This command has been globally disabled by the bot owners. Want to know why and find out when it will be back? Join the official Skyra server: <https://join.skyra.pw>',
```

Reply:
Also the message feels like downright advertising.

Reply:
Advertising to our own users... how is that bad 🤔

Reply:
It just feels... a little aggresive.

Reply:
Do you want Skyra to grow or not... In the grand scheme of things this is probably one of the least aggressive steps we have to take to ever rise to the size of Pengu.

Reply:
Note that I don't disagree with you, contrary, I agree. But I also think it's acceptable. Especially considering we don't disable commands that often and this only applies to commands that *we* disabled.

Reply:
For the record the error is thrown here: https://github.com/skyra-project/klasa/blob/settings/src/inhibitors/disabled.js